src/backend/src/services/change-requests.services.ts

```diff
@@ -104,204 +151,20 @@ export default class ChangeRequestsService {
-    if (foundCR.type === CR_Type.ACTIVATION && foundCR.activationChangeRequest && accepted) {
-      const { activationChangeRequest: actCr } = foundCR;
-      const shouldUpdateProjLead = actCr.projectLeadId !== foundCR.wbsElement.projectLeadId;
-      const shouldUpdateProjManager = actCr.projectManagerId !== foundCR.wbsElement.projectManagerId;
-      const shouldChangeStartDate =
-        actCr.startDate.setHours(0, 0, 0, 0) !== foundCR.wbsElement.workPackage?.startDate.setHours(0, 0, 0, 0);
-      const changes = [];
-
-      if (shouldUpdateProjLead) {
-        const oldPL = await getUserFullName(foundCR.wbsElement.projectLeadId);
-        const newPL = await getUserFullName(actCr.projectLeadId);
-        changes.push({
-          changeRequestId: foundCR.crId,
-          implementerId: reviewer.userId,
-          wbsElementId: foundCR.wbsElementId,
-          detail: buildChangeDetail('Project Lead', oldPL, newPL)
-        });
-      }
-
-      if (shouldUpdateProjManager) {
-        const oldPM = await getUserFullName(foundCR.wbsElement.projectManagerId);
-        const newPM = await getUserFullName(actCr.projectManagerId);
-        changes.push({
-          changeRequestId: foundCR.crId,
-          implementerId: reviewer.userId,
-          wbsElementId: foundCR.wbsElementId,
-          detail: buildChangeDetail('Project Manager', oldPM, newPM)
-        });
-      }
-
-      if (shouldChangeStartDate) {
-        changes.push({
-          changeRequestId: foundCR.crId,
-          implementerId: reviewer.userId,
-          wbsElementId: foundCR.wbsElementId,
-          detail: buildChangeDetail(
-            'Start Date',
-            foundCR.wbsElement.workPackage?.startDate.toLocaleDateString() || 'null',
-            actCr.startDate.toLocaleDateString()
-          )
-        });
-      }
-
-      changes.push({
-        changeRequestId: foundCR.crId,
-        implementerId: reviewer.userId,
-        wbsElementId: foundCR.wbsElementId,
-        detail: buildChangeDetail('status', foundCR.wbsElement.status, WBS_Element_Status.ACTIVE)
-      });
-
-      await prisma.change.createMany({ data: changes });
-
-      await prisma.wBS_Element.update({
-        where: { wbsElementId: foundCR.wbsElementId },
-        data: {
-          projectLeadId: actCr.projectLeadId,
-          projectManagerId: actCr.projectManagerId,
-          workPackage: { update: { startDate: actCr.startDate } },
-          status: WBS_Element_Status.ACTIVE
-        }
-      });
+      await this.reviewScopeChangeRequest(foundCR, reviewer, psId);
+    } else if (accepted && foundCR.type === CR_Type.STAGE_GATE) {
+      // stage gate cr
+      await this.reviewStageGateChangeRequest(foundCR, reviewer);
+    } else if (foundCR.type === CR_Type.ACTIVATION && foundCR.activationChangeRequest && accepted) {
+      // if it's an activation cr and being accepted, we can do some stuff to the associated work package
```

Comment:
move these comments to right above their corresponding if statement, and clean them up, e.g. "Stage Gate Change Request"